test/metabase/sync/analyze/fingerprint_test.clj

```diff
@@ -120,7 +121,8 @@
 (defn- field-was-fingerprinted? {:style/indent 0} [fingerprint-versions field-properties]
   (let [fingerprinted? (atom false)]
     (with-redefs [i/fingerprint-version->types-that-should-be-re-fingerprinted fingerprint-versions
-                  metadata-queries/table-rows-sample                           (constantly [[1] [2] [3] [4] [5]])
+                  qp/process-query                                             (fn [_ {:keys [rff]}]
```

Comment:
This is reasonable all things considered, although to be honest they way I've written these tests using with-redefs gross af